### ISSUE TITLE
New Error

### ISSUE THREAD

**Issue by JachinY** [dckiller51/bodymiscale#171]
Today, I find out there is always one of my two bodymiscale.entity shows unavailable, once I reload the unavailable one, it will work correctlly, but the other one will be turn to unavailable. In a word, there can be only one correct entity and always one entity will be unavailable.

**Comment by JachinY** [comment]
![image](https://github.com/dckiller51/bodymiscale/assets/42339509/39b2946d-9410-4189-9681-4e8ad71c65f2)

**Comment by supperka** [comment]
yes, i write same issue , couple weeks ago, no responds from author.

**Comment by dckiller51** [comment]
You can have several functional entities at the same time but you must store your weight and impedance in sensors other than that of the scale.

**Comment by kimm-root** [comment]
The same problem, two users each with their own card. But the data is not updated until the integration is restarted. In this case, the data is updated, but the sensors in the other user's card become unavailable.
